software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -80,9 +84,22 @@ public void launch() {
 
     @Override
     public boolean isRunning() {
-        int exitCode = executeCommandInTask(
-                getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_COMMAND),
-                getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_POWERSHELL_COMMAND), "is-running-command");
+        int exitCode = 1;
+        try {
+            exitCode = executeCommandInTask(
+                    getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_COMMAND),
+                    getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_POWERSHELL_COMMAND), "is-running-command");
+        } catch (Exception e) {
+            // If machine is restarting, then will get WinRM IOExceptions - don't propagate such exceptions
+            Throwable interestingCause = Exceptions.getFirstThrowableOfType(e, WebServiceException.class) != null ?
+                    Exceptions.getFirstThrowableOfType(e, WebServiceException.class/*Wraps Soap exceptions*/) : Exceptions.getFirstThrowableOfType(e, Fault.class/*Wraps IO exceptions*/);
+            if (interestingCause != null) {
+                LOG.warn(getEntity() + " isRunning check failed. Executing WinRM command failed.", e);
```

Comment:
I wonder how many times / how often we might get this exception logged if the VM is rebooting for several minutes.